badger/cmd/write_bench.go

```diff
@@ -249,10 +242,8 @@ func writeBench(cmd *cobra.Command, args []string) error {
 		WithCompactL0OnClose(force).
 		WithValueThreshold(valueThreshold).
 		WithNumVersionsToKeep(numVersions).
-		WithMaxCacheSize(maxCacheSize << 20).
-		WithKeepBlockIndicesInCache(keepBlockIdxInCache).
-		WithKeepBlocksInCache(keepBlocksInCache).
-		WithMaxBfCacheSize(maxBfCacheSize << 20).
+		WithBlockCacheSize(maxCacheSize << 20).
+		WithIndexCacheSize(maxBfCacheSize << 20).
```

Comment:
```suggestion
		WithBlockCacheSize(10 << 20).
		WithIndexCacheSize(10 << 20).
```

Reply:
@NamanJain8 Why do think we should hardcode the values?

Reply:
Sorry, I mean replace it with `blockCacheSize` and `indexCacheSize` respectively.

Reply:
Fixed.